packages/create-plugin/src/types.ts

```diff
@@ -0,0 +1,24 @@
+import { PLUGIN_TYPES } from './constants.js';
+
+export type CliArgs = {
+  pluginName: string;
+  pluginDescription: string;
+  orgName: string;
+  pluginType: PLUGIN_TYPES;
+  hasBackend: boolean;
+  hasGithubWorkflows: boolean;
+  hasGithubLevitateWorkflow: boolean;
+};
+
+export type TemplateData = {
+  pluginId: string;
+  packageManagerName: string;
+  packageManagerInstallCmd: string;
+  packageManagerVersion: string;
+  isAppType: boolean;
+  isNPM: boolean;
```

Comment:
do we really need this flag since we already have `packageManagerName` ?

Reply:
Due to the way NPM passes flags to run other NPM scripts it's needed for:

```
"lint:fix": "{{ packageManagerName }} run lint{{#if isNPM}} --{{/if}} --fix",
```

The alternative is to remove it in favour of duplicating the entire `lint` script and addind the extra param.